templates/plugin/write_graphite/carbon.conf.erb

```diff
@@ -17,7 +17,10 @@
   LogSendErrors <%= @logsenderrors %>
   Protocol "<%= @protocol %>"
 <%- end -%>
-<%- if scope.lookupvar('collectd::collectd_version_real') and (scope.function_versioncmp([scope.lookupvar('collectd::collectd_version_real'), '5.3']) >= 0) -%>
```

Comment:
if this was an EPP template, these `scope.lookupvar()` calls could simply be a `$`

Reply:
I know, but for consistency I kept the same format as the rest of the template (https://github.com/voxpupuli/puppet-collectd/blob/master/templates/plugin/write_graphite/carbon.conf.erb#L16 and https://github.com/voxpupuli/puppet-collectd/blob/master/templates/plugin/write_graphite/carbon.conf.erb#L20)

Reply:
aye, hence why this is just a comment, not a request for you to change anything.

let's do this in a follow up pr, tho.